Introduction/tutorials/Introduction.md

```diff
@@ -1,106 +1,404 @@
 
-- *First objective of this tutorial (It is a single sentence describing what a learner will be able to do once they have sat through the lesson. The objectives must be technical, but also theoretical, objectives. You can check [SWC lessons](http://swcarpentry.github.io/instructor-training/19-lessons/) to help you writing learning objectives.)*
-- *Second objective*
-- *Third objective*
-- *...*
+- *Familiarize yourself with the basics of Galaxy*
+- *Learn how to obtain data from external sources*
+- *Learn how to run tools*
+- *Learn how histories work*
+- *Learn how to create a workflow*
+- *Learn how to share your work*
 
 :heavy_check_mark: ***Requirements***
 
-- *Galaxy introduction*
-- *Second requirement*
-- *Third requirement*
-- *...*
+- *No prior knowledge is required*
 
-:hourglass: ***Time estimation*** *1d/3h/6h*
+
+:hourglass: ***Time estimation*** *1-1,5h*
 
 # Introduction
 
-General introduction about the topic and then an introduction of the tutorial (the questions and the objectives). It is nice also to have a scheme to sum up the pipeline used during the tutorial. The idea is to give to trainees insight into the content of the tutorial and the (theoretical and technical) key concepts they will learn.
+This practical aims to familiarize you with the Galaxy user interface. It will teach you how to perform basic tasks such as importing data, running tools, working with histories, creating workflows, and sharing your work.
+
+# Getting started
+
+In this very simple example analysis we will introduce you to the basics of Galaxy. In this tutorial you will perform the following tasks:
+
+- Getting data from UCSC
+- Running simple data manipulation tools on this data
+- Displaying your data at the UCSC main genome viewer
+- Understanding Galaxy’s tool and history system
+- Extracting a workflow from your analysis history
+- Running a workflow on new data
+- Sharing your work with others
+
+Suppose you get the following question:
+
+*Mom (or Dad) ... Which coding exon has the highest number of single nucleotide polymorphisms on chromosome 22?*
+
+You think to yourself Wow! This is a simple question ... I know exactly where the data is (at UCSC) but how do I actually compute this? The truth is, there is really no straightforward way of answering this question in a time frame comparable to the attention span of a 7-year-old. Well ... actually there is and it is called Galaxy. So let’s try it...
+
+## Preparations
+
+Time to get started, browse to your Galaxy instance and log in or register. The Galaxy interface consist of three main parts. The available tools are listed on the left, your analysis history is recorded on the right, and the middle pane will show the tools and datasets.
+
+![](../images/galaxy-interface.png)
+
+:pencil2: ***Hands on!***
+
+1. Make sure you have an empty analysis history. To start a new history, click on the **gear icon** at the top of the history pane, and select the option **Create New** from the menu.
```

Comment:
pane is a word too, means basically the same as panel (but if peopole think panel is more widely understood I can change it)